mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -937,4 +937,110 @@ Feature: Loans tests
     Given path '/_/proxy/tenants/' + tenant + '/timers'
     And request '{"id":"mod-circulation_6","routingEntry":{"unit": "minute", "delay":"30"} }'
     When method PATCH
-    Then status 204
\ No newline at end of file
+    Then status 204
+
+  Scenario: When patron has exceeded their Patron Group Limit for 'Maximum number of items charged out', patron is not allowed to borrow items per Conditions settings
+
+    * def extItemBarcode1 = 'FAT-1019IBC - 1'
+    * def extItemBarcode2 = 'FAT-1019IBC - 2'
+    * def extUserBarcode = 'FAT-1019UBC'
+
+    # location and service point setup
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostLocation')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostServicePoint')
+
+    # post an items
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostInstance')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostHoldings')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: #(extItemBarcode1) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: #(extItemBarcode2) }
+
+    # post a group and an user
+    * def extUserId = call uuid1
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup') { extUserGroupId: #(groupId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(extUserId), extUserBarcode: #(extUserBarcode) }
+
+    # get the interested condition id, Maximum number of items charged out
+    Given path 'patron-block-conditions'
+    When method GET
+    Then status 200
+    * def conditions = karate.sort(response.patronBlockConditions, (condition) => condition.name)
+    * def conditionId = conditions[0].id
+
+    # set block actions, renewals and requests to the condition
+    * def blockMessage = 'You have blocked!'
+    * def conditionUpdateRequest = { id: #(conditionId), blockBorrowing: false, blockRenewals: true, blockRequests: true, message: #(blockMessage), name: 'Maximum number of items charged out', valueType: 'Integer' }
+    Given path 'patron-block-conditions', conditionId
```

Comment:
This call we use two times. Pls, consider to move it to initData.feature

Reply:
fixed, thanks